app-foundation/3-networks-extension/network_prepare.sh

```diff
@@ -20,6 +20,8 @@ parent_dir=$( dirname "$(pwd)" )
 if [[ ! -d "$parent_dir/3-networks" ]]; then
     git clone --depth 1 --filter=blob:none https://github.com/terraform-google-modules/terraform-example-foundation example-foundation
     mv example-foundation/3-networks/ "$parent_dir"
+    mv example-foundation/build/cloudbuild-tf-* "$parent_dir"/../build
```

Comment:
Why are we moving the `cloudbuild-tf-*` and wrapper files? These are in the networking deploy instructions

Reply:
this will make it easier for the end user to follow instructions as the idea is starting from 3-networks the new working env is example-foundation-app, this will reduce the dependency on still keeping example-foundation cloned locally for files needed

Reply:
discussed offline, we are copying for better ux when instructions say cp from build (example-foundations build dir vs example-app build dir)